signer/tests/integration/setup.rs

```diff
@@ -847,13 +860,31 @@ impl TestSweepSetup2 {
 
     /// This function generates a sweep transaction that sweeps in the
     /// deposited funds and sweeps out the withdrawal funds in a proper
-    /// sweep transaction, that is also confirmed on bitcoin.
-    pub fn submit_sweep_tx(&mut self, rpc: &Client, faucet: &Faucet) {
+    /// sweep transaction, it broadcasts this transaction to the bitcoin
+    /// network.
+    pub fn broadcast_sweep_tx(&mut self, rpc: &Client) {
         // Okay now we try to peg-in the deposit by making a transaction.
         // Let's start by getting the signer's sole UTXO.
         let aggregated_signer = &self.signers.signer;
         let signer_utxo = aggregated_signer.get_utxos(rpc, None).pop().unwrap();
 
+        // Well we want a BitcoinCoreClient, so we create one using the
+        // settings. Not, the best thing to do, sorry. TODO: pass in a
+        // bitcoin core client object.
```

Comment:
This will all die "soon" with some integration test improvements anyway, right? ;)